Sources/TuistSupport/CoreDataVersionExtractor.swift

```diff
@@ -3,6 +3,15 @@ import TSCBasic
 
 /// Extract version from .xccurrentversion file.
 public final class CoreDataVersionExtractor {
+    /// Returns whether or not the provided core data model has versions
+    ///
+    /// - Parameter path: absolute path to Model.xcdatamodel
+    ///
+    /// - Returns: Whether or not the xcdatamodel has versions
+    public static func isVersioned(at path: AbsolutePath) -> Bool {
+        FileManager.default.fileExists(atPath: path.appending(component: ".xccurrentversion").url.path)
```

Comment:
minor: you can leverage `FileHandler.shared.exists()` which supports the `AbsolutePath` type and is mockable in tests :)

Reply:
thanks!